babble-backend/controllers/posts-controller.js

```diff
@@ -125,6 +125,9 @@ function search(req, res) {
         let inComments = false;
         const comments = post.comments;
         for (i = 0; i < comments.length; i++) {
+            if(!comments[i].body || !comments[i].time){
+                continue;
+            }
```

Comment:
This looks good, but it'll be better if we never allow undefined comment bodies and time. This comment is more of a reminder for backend to make that change. I'll take care of it.